data_tools/buildhub/utils.py

```diff
@@ -0,0 +1,22 @@
+from os.path import basename
+
+
+def build_record_id(url):
+    # Generate an ID that looks like that: firefox_53.0b9_linux-i686_en-US
+    parts = basename(url).split('.')
+    url_parts = url.split('/')
+
+    product_name, major_version = parts[0].split('-')
```

Comment:
We have test that cover this. Here we are taking the name in the archive name not the first parts of the URL.